specification/resources/resource-manager/Microsoft.Resources/stable/2024-07-01/resources.json

```diff
@@ -6465,7 +6559,7 @@
         "status": {
           "type": "string",
           "description": "Status of the What-If operation."
-        },
+        },        
```

Comment:
at a guess, this trailing whitespace here might be the reason for prettier being unhappy

Reply:
Weird,,,